pages/HomePage.ts

```diff
@@ -97,5 +101,45 @@ export class HomePage {
         await this.ccLink.click();
         await expect(this.page.url()).toContain(this.ccUrl);
     }
+
+        async verifyInformationTitles(): Promise<void> {
+            const urls = [
```

Comment:
It should be changed. Create constants.ts and add those cons as "export const URL"

Reply:
Done